### DISCUSSION TITLE
wold it posible?

### DISCUSSION THREAD

**Discussion by MatejMagat305** [raysan5/raylib#3059] [Q&A] [answered]

**Comment by MatejMagat305** [comment]
so sorry, I notice own java is still needed ..., but I want to discuss about https://github.com/cnlohr/rawdrawandroid, this create smaller binary ( 45 KB ), is it not suitable or already it is same code, but raylib is simply so "big"? (only hypothetically) if it the same, then could somebody explain me please, why raylib need java and rawdrawandroid not?

**Comment by ndytts** [comment, marked as answer]
Strictly speaking, you don't need java to create an apk with raylib.

You just have to compile the src makefile for android and then with that library, you just follow the steps in the makefile that the project [rawdrawandroid](https://github.com/cnlohr/rawdrawandroid) has, of course, with some minor changes.

I did it and the resulting apk with raylib+raygui was 300kb (static), and the libmain.so (unzipped) is 626kb.
